package/yast2-samba-server.spec

```diff
@@ -30,7 +30,6 @@ Source0:        %{name}-%{version}.tar.bz2
 BuildRequires:  libsmbclient-devel
 BuildRequires:  perl-Crypt-SmbHash
 BuildRequires:  perl-X500-DN
-BuildRequires:  perl-XML-Writer
```

Comment:
I miss dropping the BuildRequires on yast2-testsuite (looks like we always forget one of the two).

Reply:
Weird. I would swear I deleted it... 

Anyway, fixed :)